tools/tests/form-control/utils/constants.js

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Coherent Labs AD. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export const SERVER_PORT = 12345;
+export const RESPONSE_CONTAINER_ID = 'form-response';
+
+export const formsIds = {

```

Comment:
```suggestion
export const FORMS_IDS= {

```
?

Reply:
Well [google style guide](https://google.github.io/styleguide/jsguide.html#features-objects-enums) says that it should be with CamelCase :D